kernel/devices/Rtc.cpp

```diff
@@ -0,0 +1,62 @@
+#include <devices/Rtc.h>
+#include <Cpu.h>
+#include <Log.h>
+
+namespace Kernel::Devices 
+{
+
+    uint8_t readRtcRegister(uint8_t rtcRegister)
```

Comment:
Please use the same coding style as the rest of the code: function names use PascalCase.